CHANGELOG.md

```diff
@@ -156,6 +158,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
       returns temporal info for CMR ISO 19115 SMAP XML files.
     - Updated `@cumulus/cmrjs/cmr-utils.isCmrFilename()` to include
       `isISOFile()`.
+- **CUMULUS-2532**
+  - Changed integration tests to use `api-client/granules` functions as opposed
+    to `granulesApi` from `@cumulus/integration-tests`.
+- **CUMULUS-2301**
+  - Updated `getAsyncOperation` to read from PostgreSQL database instead of
```

Comment:
This note should go under the "Changed" section

Reply:
[`2c9bd18` (#2294)](https://github.com/nasa/cumulus/pull/2294/commits/2c9bd186888194930101931e267116772c417d7c)